### ISSUE TITLE
Support form fillable PDFs in Journal Pages

### ISSUE THREAD

**Issue by farling42** [farling42/fvtt-pdf-pager#76]
One user on Discord wants to have a form fillable PDF stored in a journal page, and just store the entered data directly on that page.
![image](https://github.com/farling42/fvtt-pdf-pager/assets/17485384/85a191b7-3379-4303-8da1-699ad79910ee)
Today = 14/Oct/2023

**Comment by farling42** [comment]
The current work-around is to set the PDF Code to "TEST" and get the UUID of the journal page, and set a macro:
```js
ui.pdfpager.openPDFByCode("TEST", {uuid: "JournalEntry.Kcy32XYut0HhHu9p.JournalEntryPage.uwmQZAyxKloxtiyz"})
```

**Comment by farling42** [comment]
Implemented in 0.49.0
